example/grackle/README

```diff
@@ -0,0 +1,14 @@
+First-time GRACKLE installation guide (on calab-ntu gpu-clusters as an example):
+========================================
+1. First download the source code by executing "git clone https://github.com/grackle-project/grackle"
+2. Next navigate to "cd grackle". The required UVB interpolation tables can be downloaded via "git submodule update --init"
+3. Next navigate to "./configure; cd src/clib"
+4. Copy into the config files "Make.mach.eureka-intel" or "Make.mach.spock-intel" (from the directory "<your_gamer_directory>/example/grackle/") and change the prefix: "MACH_INSTALL_PREFIX = <your_grackle_directory>/grackle_spock" in "Make.mach.spock-intel" to the path you wish to install the package
+5. Next execute "make clean; make machine-spock-intel" or "make clean; make machine-eureka-intel"
+6. Next execute "make show-config" and "make precision-32" (single precision is more compatible with gaming GPUs on eureka and spock) and "make omp-on" (GAMER does utilize OpenMP)
+7. Next execute "make save-config-gamer" (save this GAMER-spock specific config setting) and "make load-config-gamer" (load this GAMER-spock specific config setting)
+8. Lastly for code compilation, execute "make; make install"
+9. To compile the corresponding gamer executable:
+    (1) Add the path "GRACKLE_PATH    <your_grackle_directory>/grackle/grackle_spock_single_omp" to "<your_gamer_directory>/configs/spock_intel.config"

```

Comment:
The path `GRACKLE_PATH` is different from step 4. You might want it to be `<your_grackle_directory>/grackle_spock`.